src/main/scala/gitbucket/core/api/JsonFormat.scala

```diff
@@ -20,7 +20,7 @@ object JsonFormat {
       { case x: Date => JString(parserISO.print(new DateTime(x).withZone(DateTimeZone.UTC))) }
     )
   ) + FieldSerializer[ApiUser]() +
-    FieldSerializer[ApiPullRequest]() +
+    FieldSerializer[ApiPullRequest](FieldSerializer.ignore("gitbucket$core$service$AccountService$$logger")) +
```

Comment:
It's not necessary if an instance of `Account` of assignee is given by caller of `ApiPullRequest.apply()`.